2200074/1204.py

```diff
@@ -0,0 +1,13 @@
+from collections import Counter
+
+T = int(input())
+
+for test_case in range(1, T + 1):
+    case_number = int(input())
+    scores = list(map(int, input().split(',')))
```

Comment:
```suggestion
    scores = list(map(int, input().split()))
```
입력 값이 쉼표가 아닌 공백으로 구분되어 있으므로 위와 같이 수정해주셔야 해요!
또한 문제 조건 중 '최빈수가 여러 개라면 그 중 가장 큰 값을 출력한다' 라는 말이 있습니다! 테스트 케이스에서는 해당 경우가 주어지지 않았지만, 주어졌을 경우 어떻게 처리해야 할지도 생각해보시면 좋을 것 같습니다. 😄